src/main/scala/akka/persistence/cassandra/ClusterBuilder.scala

```diff
@@ -0,0 +1,48 @@
+package akka.persistence.cassandra
+
+import com.datastax.driver.core.policies.{DCAwareRoundRobinPolicy, TokenAwarePolicy}
+import com.datastax.driver.core.{Session, Cluster, QueryOptions, SSLOptions}
+
+import scala.collection.JavaConverters._
+
+object ClusterBuilder {
```

Comment:
The code was just repeated too often and it did not feel right when this logic was in config.